test-result-summary-client/src/PerfCompare/PerfCompare.jsx

```diff
@@ -81,29 +83,49 @@ export default class PerfCompare extends Component {
     }
 
     handleGetPerffarmRuns = async () => {
-
-        this.setState(
+    	let resBenchmarkRuns = await fetch(`/api/getPerffarmRunCSV?${stringify(this.state.selectedRuns)}`, {
```

Comment:
Why have you changed the order by moving this get request before setState()?

Reply:
You are right, changed it back already.